browsergym/core/src/browsergym/core/action/highlevel.py

```diff
@@ -514,6 +514,56 @@ def to_python_code(self, action):
         # return the constructed python code
         return python_code
 
+    def to_tool_description(self, api="openai") -> list[dict]:
+        """
+        Translates actions to tool descriptions following the OpenAI API format.
+
+        Returns:
+            A list of tool descriptors.
+        """
+        schema_keys = {
+            "openai": "parameters",
+            "anthropic": "input_schema",
+        }
+        schema = schema_keys.get(api, "parameters")
+        tools = []
+        for tool_name, action in self.action_set.items():
+            # Parse the signature to extract parameter names and types
+            parameters = {"type": "object", "properties": {}, "required": []}
```

Comment:
Isolate from here into a function get_tool(name, action) -> tool and unit test it.